test/testutil/driver.c

```diff
@@ -120,16 +117,7 @@ static void finalize(int success)
     if (success)
         ERR_clear_error();
     else
-        ERR_print_errors_cb(err_cb, NULL);
-}
-
-static void helper_printf_stdout(const char *fmt, ...)
-{
-    va_list ap;
-
-    va_start(ap, fmt);
-    test_vprintf_stdout(fmt, ap);
-    va_end(ap);
+        TEST_openssl_errors();
```

Comment:
Let's ask ourselves why was it appropriate to stick to `ERR_*` in both if cases? Because it's actually not inappropriate to adhere to same ["visual"] patterns as in normal code, isn't it? Or in other words it's not inappropriate for test programs to look more like application code, or the way we'd like to see application code look. [For this reason I'm not actually fond of **excessive** use of TEST_this_n_that.]